wsgi.py

```diff
@@ -4,9 +4,13 @@
 import os
 
 
+_application = create_web_app()
+
+
 def application(environ, start_response):
     """WSGI application factory."""
+    # Apache passes config from SetEnv directives via the request environ.
     for key, value in environ.items():
-        os.environ[key] = str(value)
-    app = create_web_app()
-    return app(environ, start_response)
+        if key in _application.config:
```

Comment:
This might not be good enough. I ran into an issue where ``SERVER_NAME`` was getting passed in uwsgi, and it mucked things up. See what you think